experimental/README.md

```diff
@@ -11,7 +11,9 @@ please feel free to provide any feedback on these features you wish.
 
 ## Install Docker experimental
 
-Unlike the regular Docker binary, the experimental channels is built and updated nightly on TO.BE.ANNOUNCED. From one day to the next, new features may appear, while existing experimental features may be refined or entirely removed.
+Unlike the regular Docker binary, the experimental channels is built and updated nightly on TO.BE.ANNOUNCED. From one day to the next, new features may appear, while existing experimental features may be refined or entirely removed. 
```

Comment:
`channels __are__ built` :)

I wonder what `TO.BE.ANNOUNCED` is - I'm running experimental builds from `https://experimental.docker.com` - :)